packages/editor/src/components/post-locked-modal/index.js

```diff
@@ -30,17 +30,30 @@ class PostLockedModal extends Component {
 	}
 
 	componentDidMount() {
```

Comment:
Should we consider the `heartbeat.error` event/hook as well? I'm not that familiar with the heartbeat API so I don't know in what specific situation that would be fired that `heartbeat.tick` doesn't cover. At first sight, it seems sensible. If that makes sense, I'm happy if we do that in a follow-up PR and I can prepare one.

Reply:
> Should we consider the `heartbeat.error` event/hook as well? I'm not that familiar with the heartbeat API so I don't know in what specific situation that would be fired that `heartbeat.tick` doesn't cover. At first sight, it seems sensible. If that makes sense, I'm happy if we do that in a follow-up PR and I can prepare one.

Maybe, but as you alluded, I'm more concerned on an in-place-compatible refactoring here; no additions of behavior.